.nvmrc

```diff
@@ -1 +1 @@
-4.2.0
+4.8.4
```

Comment:
maybe move to 6 or 8? 8 will be LTS in October

Reply:
I like the idea of having the default still be the oldest supported version.